recipes/e3sm_to_cmip/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "e3sm_to_cmip" %}
+{% set version = "1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/E3SM-Project/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: a23d7440bab9950f6fef8a6e5fe60741e4e5be2a2421d37ca9a1739dab1deef1
+
+build:
+  skip: True  # [win or py<36]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

you may add the ``entry_points`` and it should work on windows as well
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  entry_points:
    - e3sm_to_cmip = e3sm_to_cmip.__main__:main
```

---

Oops, thanks!

---

After that, I think you can remove that selector to skip ``win`` and you might selector  the python versions which you want to on host and run requirements
After all that you can turn this recipe to be ``noarch: python``
https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

---

We could do that but I don't think we want to give the misimpression that Windows is supported.